experiments/compositions/expanders/cel-expander/config/release/kustomization.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Adds namespace to all resources.
+namespace: composition-system
+namePrefix: composition-
+resources:
+- ../crd
+- ../expanders
+- ../expander-versions
+
+patches:
+- patch: '[{"op": "replace", "path": "/spec/template/spec/containers/0/image",
+    "value": "gcr.io/krmapihosting-release/expander-cel:v0.0.1"}]'
+  target:
+    kind: Deployment
+    name: cel-v0.0.1
```

Comment:
Similar here, seems the expander version is hardcoded?

Reply:
This is ok. We can override this once we add a build pipeline.